pkg/syncer/util.go

```diff
@@ -84,7 +84,7 @@ func getInlineMigratedVolumesInfo(ctx context.Context, metadataSyncer *metadataS
 					log.Warnf("FullSync: Failed to get VolumeID from volumeMigrationService for volumePath: %s with error %+v", volume.VsphereVolume.VolumePath, err)
 					continue
 				}
-				inlineVolumes[volume.VsphereVolume.VolumePath] = volumeHandle
+				inlineVolumes[volumeHandle] = volume.VsphereVolume.VolumePath
```

Comment:
Why are we swapping key and value?

Reply:
Yeah, this is needed for fast lookup here: https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/syncer/fullsync.go#L505
Also, this is a miss from earlier PRs